cmd/encryption-v1.go

```diff
@@ -141,55 +149,79 @@ const (
 	ServerSideEncryptionSealedKey = ReservedMetadataPrefix + "Server-Side-Encryption-Sealed-Key"
 )
 
-// SSESealAlgorithmDareSha256 specifies DARE as authenticated en/decryption scheme and SHA256 as cryptographic
-// hash function.
-const SSESealAlgorithmDareSha256 = "DARE-SHA256"
+const (
+	// SSEIVSize is the size of the randomly chosen IV used to derive the key-encryption-key.
+	SSEIVSize = 32
+
+	// DAREPayloadSize is the max. size of the package payload for a DARE 2.0 package.
+	DAREPayloadSize = 1 << 16
+
+	// DAREOverhead is the overhead (header + tag) added by encrypting one payload using DARE 2.0.
+	DAREOverhead = 32
+
+	// SSESealAlgorithmDareSha256 specifies DARE as authenticated en/decryption scheme and SHA256 as
+	// cryptographic hash function.
+	SSESealAlgorithmDareSha256 = "DARE-SHA256"
+)
+
+var (
+	// SSECustomerHeaders can parse regular SSE-C requests.
+	SSECustomerHeaders = sseCustomerHeaders{
+		algorithm: SSECustomerAlgorithm,
```

Comment:
I think it makes sense to remove `SSECustomerAlgorithm` constant or variable here and replace it with the real string. The same goes for the next few lintes.

Reply:
Yes, I agree we can.

Reply:
It's not just used for parsing - the multipart logic uses them to. I can address in separate PR.